tema14Ianuarie/src/com/sci/bv/tema14Ianuarie/DailySales.java

```diff
@@ -0,0 +1,49 @@
+package com.sci.bv.tema14Ianuarie;
+
+/**
+ * This Class stores products sold on a particular day (Daily Sales).
+ * In this Class are storing the product's ID, sold quantity and date of sale.
+ */
+public class DailySales {
```

Comment:
Class names should be singular, so this would be "DailySale". 
Also... what makes this a DailySale as opposed to a regular Sale? It just holds all the information needed to identify a sale: which product (id), how much (qty) and when.